server/routerlicious/packages/routerlicious-base/src/nexus/services/storageNameAllocator.ts

```diff
@@ -0,0 +1,13 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IStorageNameAllocator, ITenantConfigManager } from "@fluidframework/server-services-core";
+
+export class StorageNameAllocator implements IStorageNameAllocator {
+	public constructor(private readonly tenantConfigManager: ITenantConfigManager) {}
+	public async assign(tenantId: string, documentId: string): Promise<string> {
```

Comment:
I don't think this is used in nexus flow either. please remove it after confirming.

Reply:
It is actually used to create a storage object that ends up being passed to configureWebSocketServices method. However, in there, it is not used for anything and I actually see 2 other parameters that remain unused.

This cleanup is best saved for a later PR as configureWebSocketServices is used by both Alfred and Nexus.